source/stylesheets/_landing.scss

```diff
@@ -125,68 +138,67 @@
   }
 }
 
-.actionable-sections {
-  padding: 0 $side-padding;
+.upcoming,
+.mailing-list {
+  background: $white;
+  padding: 3em $side-padding;
+}
+
+.upcoming {
+  border-bottom: 3px $blue-light solid;
+  margin: auto;
 
   @include media($medium-screen) {
-    @include clearfix;
-    @include outer-container;
-    position: relative;
+    @include display(flex);
+    @include padding(null 2.5em);
+    border: 3px $blue-light solid;
+    border-radius: $base-border-radius;
+    margin: 20vh auto;
+    max-width: 44em;
   }
 
-  &:before {
-    @include media($medium-screen) {
-      @include position(absolute, null 0 null 0);
-      @include size(1px 100%);
-      background-color: $base-border-color;
-      content: "";
-      display: block;
-      margin: auto;
-    }
-  }
+  iframe {
+    width: 100%;
 
-  section {
     @include media($medium-screen) {
-      padding-bottom: 5em;
-      padding-top: 5em;
+      max-width: em(300);
+      order: 2;
     }
   }
 }
 
-.next-workshop {
+.workshop {
+  margin: 1.5em auto 0 auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `1.5em auto 0` instead of `1.5em auto 0 auto`